specification/ai/Azure.AI.Projects/agents/files/routes.tsp

```diff
@@ -110,13 +110,13 @@ op getFile is Azure.Core.Foundations.Operation<
 #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "not yet versioned"
 @get
 @route("/files/{fileId}/content")
-@doc("Returns information about a specific file. Does not retrieve file content.")
+@doc("Retrieves the raw content of a specific file.")
 op getFileContent is Azure.Core.Foundations.Operation<
   {
     @doc("The ID of the file to retrieve.")
     @path
```

Comment:
Nit, for later cleanup: I don't think the "@path" are needed (I see you use them everywhere in Agents). I don't see this decorator defined in https://typespec.io/docs/standard-library/built-in-decorators/. I did not need it for example for my `op getConnection is Azure.Core.Foundations.Operation<`

Reply:
ok, thanks, will check that later when doing more doc updates.